templates/patterns/listing/listing.ui_patterns.yml

```diff
@@ -1,52 +1,53 @@
+      description: 'Selects the number of columns of the list:'
       options:
-        default-1-col: "default-1-col"
-        default-2-col: "default-2-col"
-        default-3-col: "default-3-col"
-        highlight-1-col: "highlight-1-col"
-        highlight-2-col: "highlight-2-col"
-        highlight-3-col: "highlight-3-col"
-      preview: "default-3-col"
+        default-1-col: 'default-1-col (default)'
+        default-2-col: 'default-2-col'
+        default-3-col: 'default-3-col'
+        highlight-1-col: 'highlight-1-col'
+        highlight-2-col: 'highlight-2-col'
+        highlight-3-col: 'highlight-3-col'
+      preview: 'default-3-col'
     title_tag:
-      type: "textfield"
-      label: "Title tag"
-      description: "Tag that will be used for the title of the card. It will be 'h4' by default."
-      preview: "h5"
+      type: textfield
+      label: 'Title tag'
+      description: 'Tag that will be used for the title of the card. It will be "h4" by default.'
+      preview: 'h5'
     alignment:
-      type: "select"
-      label: "Alignment"
-      description: "The alignment of the image in the default layout cards."
+      type: select
+      label: 'Alignment'
+      description: 'The alignment of the image in the default layout cards:'
       options:
-        default: "[empty]"
-        start: start
-        end: end
-        center: center
-        baseline: baseline
-        stretch: stretch
-      preview: "start"
+        default: '[empty] (default)'
+        start: 'start'
+        end: 'end'
+        center: 'center'
+        baseline: 'baseline'
+        stretch: 'stretch'
+      preview: 'start'
   fields:
     items:
-      type: "array"
-      label: "List items"
-      description: "List of items"
+      type: array
+      label: 'Items'
+      description: 'Collection of items that will be displayed as a list. Each item can hold an image and/or a date. When the date is reported, then a date block is used to present it. Otherwise an image is shown in the card.'
       preview:
         - title:
             content:
               '#markup' : '<a href="/example">Card title 1</a>'
-            tag: "h5"
+            tag: 'h5'
           text:
-            content: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus ut ex tristique, dignissim sem ac, bibendum est. Sed vehicula lorem non nunc tincidunt hendrerit. Nunc tristique ante et fringilla fermentum."
-            classes: "mb-2"
+            content: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus ut ex tristique, dignissim sem ac, bibendum est. Sed vehicula lorem non nunc tincidunt hendrerit. Nunc tristique ante et fringilla fermentum.'
+            classes: 'mb-2'
           image:
-            path: "https://picsum.photos/seed/1002/600/400"
-            alt: "alt img"
-            position: "top"
+            path: 'https://placeimg.com/600/400/people'
+            alt: 'alt img'
+            position: 'top'
```

Comment:
remove this as we do it in the prerocess, doube check all these preview values after refactor